code/game/machinery/portable_turret.dm

```diff
@@ -584,8 +584,8 @@ GLOBAL_LIST_EMPTY(turret_icons)
 		return TURRET_NOT_TARGET
 
 	if(check_synth)	//If it's set to attack all non-silicons, target them!
-		if(L.lying)
-			return lethal ? TURRET_SECONDARY_TARGET : TURRET_NOT_TARGET
```

Comment:
people could just crawl at turrets and it would ignore them as if they were lying, and it wasn't lethal, it wouldn't consider them a target.

Reply:
Can we implement some system to check who is purposefully crawling, as opposed to involuntarily stunned people? Otherwise this is gonna cause some issues with it not retargeting when it should.
Another solution that part-covers this would be having it not target dead people here, as otherwise it will just blast them forever. A niche situation, but one to consider nontheless.